src/main/resources/default-lintConfig.xml

```diff
@@ -136,4 +136,10 @@
 
     <!-- Enable checking for unused ids -->
     <issue id="UnusedIds" severity="info"/>
+
+    <!-- Suppress illegal package error, see https://github.com/square/okio/issues/58 -->
+    <issue id="InvalidPackage">
+        <ignore regexp="okio-.*.jar"/>
+        <ignore regexp="retrofit-.*.jar"/>
```

Comment:
while are you are at it, can you also add `<ignore regexp="requery-.*jar"/>`?  Perhaps we should just ignore all InvalidPackage warnings from 3rd party libraries since it doesn't seem to be an issue and there's not much we can do about it anyway